x/leveragelp/keeper/msg_server_open_test.go

```diff
@@ -281,7 +283,7 @@ func (suite *KeeperTestSuite) TestOpen_PoolWithBaseCurrencyAsset() {
 			&types.MsgOpen{
 				Creator:          addresses[0].String(),
 				CollateralAsset:  ptypes.BaseCurrency,
-				CollateralAmount: sdk.NewInt(1_000_000_000_000),
+				CollateralAmount: sdk.NewInt(1_000_000_000_000_000_000),
 				AmmPoolId:        1,
 				Leverage:         sdk.MustNewDecFromStr("2.0"),
 				StopLossPrice:    sdk.MustNewDecFromStr("50.0"),
```

Comment:
@cryptokage1996 can you add a unit test that would trigger the `ErrMaxLeverageLpExists` error

Reply:
@cryptokage1996 ok make sense thanks for clarifying